tools/theme/src/Theme.ts

```diff
@@ -662,3 +442,90 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
         }
     }
 }
+
+function warnDeprecatedSystem(instance: Theme, value: SystemVariant): void {
+    if (window.__swc.DEBUG) {
+        window.__swc.warn(
+            instance,
+            'property theme in <sp-theme> has been deprecated. Please use system instead like this <sp-theme system="spectrum"/>',
+            'https://opensource.adobe.com/spectrum-web-components/tools/themes/#deprecation',
+            { level: 'deprecation' }
+        );
+        if (value === 'spectrum-two') {
```

Comment:
This reads weird to me in that if you are on `spectrum-two` you'd get two warnings, one saying deprecated and one saying beta? Am I reading that right? Beta seems like a long term possibility so I wonder if it makes sense to just create a `warnBetaSystem` which would make the call site clearer too.

Reply:
I does look to me like this would get called even if they used `system` instead of `theme` though, so probably worth breaking it a part? Or something?

Reply:
Oops, you are right. After moving this debugging code around it ended up becoming messy. I have now cleaned it up, so it now only warns once about the deprecated use of `theme`.
Following your suggestion, I also created a dedicated beta warning (for S2) that we can remove if we want. Do we want to remove this beta warning? cc: @Rajdeepc

Reply:
Yes it make sense to remove the beta warning for now! We will add this step by step once we start rolling things for S2. Thanks for doing this! Appreciate it